src/rpc/Counters.cpp

```diff
@@ -76,10 +79,11 @@ Counters::report()
         counters["forwarded"] = std::to_string(info.forwarded);
         counters["duration_us"] = std::to_string(info.duration);
 
-        obj[method] = std::move(counters);
+        rpc[method] = std::move(counters);
     }
+    obj["work_queue"] = workQueue_.get().report();
```

Comment:
Do we have a JS stringy for this? i suppose same question for the lines above.

Reply:
Not for this, but for some of the others. It's a macro that wraps fields defined in rippled. We should add some sort of mechanism to define fields just for clio as well.